starters/vue3-xstate-css/src/views/HomeView.vue

```diff
@@ -1,9 +1,35 @@
-<script setup lang="ts">
-import TheWelcome from '../components/TheWelcome.vue';
-</script>
+<script setup lang="ts"></script>
 
 <template>
   <main>
-    <TheWelcome />
+    <div class="title">
+      <h1>Vue 3 + XState + CSS</h1>
+    </div>
+    <div class="links">
+      <RouterLink to="/counter">See button example component</RouterLink>
+      <RouterLink to="/fetch">See data fetching example component</RouterLink>
+    </div>
   </main>
 </template>
+
+<style scoped>
+.title {
+  width: 80%;
+  padding: 1% 2%;
+  margin: 0 auto 5%;
+  border-radius: 4px;
+  background: var(--lightBlue);
+}
+
+h1 {
+  color: var(--white);
+  text-align: center;
+}
+
+.links {
```

Comment:
This looks like styling that might be shared across multiple pages, which wouldn't be a `scoped` style. In which case, I'd suggest either moving this to a separate file or removing the `scoped.`

Reply:
The `.links` here is specifically for the home page, so the two example links sit on top of each other. It is meant to be specific to this page only.